ports/nrf/README.md

```diff
@@ -55,6 +55,20 @@ Alternatively the target board could be defined:
 
      make BOARD=pca10040
      make flash
+
+## Compile without LTO enabled
+
+As a space optimization, LTO (Link Time Optimization) has been enabled on all
+targets in the nrf-port. The `-flto` linker flag can be toggled easily by using
+the argument LTO when building. The example below shows how to disable LTO for
+the compilation:
+
+    make BOARD=pca10040 LTO=0
+
+**Note**: There has been several issues with use of LTO in conjunction with
```

Comment:
One more: "There have been several ..." (has->have)

Reply:
Fixed, rebased and force-pushed.

Reply:
Looks good, thanks!